### ISSUE TITLE
PetalPoco.Page and DISTINCT Incorrect syntax

### ISSUE THREAD

**Issue by eman1986** [CollaboratingPlatypus/PetaPoco#175]
Hi,

It seems when I try to use a query using .Page that has the DISTINCT keyword it throws an error:

Incorrect syntax near '*'

The query I'm using executes with no errors when I run it from SQL Studios so I'm curious what the problem could be.

**Comment by davearmitage123** [comment]
Yeah I am getting the same error. Does anyone know if there is an update on this?

**Comment by arhandres** [comment]
Is there something about this?
